uber/templates/registration/form.html

```diff
@@ -178,15 +178,6 @@
     </div>
 </div>
 
-<!-- TODO: figure this out, maybe
-{% if attendee.badge_type == c.SUPPORTER_BADGE %}
-    <tr>
-        <td> Affiliate: </td>
-        <td> <input type="text" name="affiliate" value="{{ attendee.affiliate|default:"" }}" /> </td>
-    </tr>
-{% endif %}
--->
-
```

Comment:
This commented out code has been sitting here for awhile, so I deleted it while I was looking at the reg form anyway.